CMakeLists.txt

```diff
@@ -267,7 +267,7 @@ if ( ENABLE_TESTS )
       get_filename_component ( TESTNAME "${VALID_JSON}" NAME )
       add_test ( NAME validate-${TESTNAME}
 	WORKING_DIRECTORY "${DATA_DIR}/inputs"
-	COMMAND ${JSONLINT} "--allow=nonescape-characters" "${VALID_JSON}" )
+	COMMAND ${JSONLINT} "--allow=nonescape-characters --allow non-portable" "${VALID_JSON}" )
```

Comment:
It looks like this change may be causing additional failures... Maybe it is a problem with the file encoding? Are emoji like the one you included utf8 or utf16? I doubt the Fortran ISO 10646 support is actually supporting utf16, heck it probably isn't even really supporting utf8. (I am under the impression that some/most compilers store individual characters in 4 bytes--for memory alignment reasons--and it's possible that they are still doing this with ISO10646, b/c, IIRC, this is just ucs4... but I could be wrong)

It looks like jsonlint will take -e to specify explicitly the input/output encoding codec. You could try `-e utf8` or `-e utf16` to resolve the problem. Also, running the test on a local computer after `pip install --upgrade git+https://github.com/dmeranda/demjson.git && jsonlint --version` should give you equivalent results.